tests/unit/tooltip/tooltip_core.js

```diff
@@ -220,3 +220,35 @@ asyncTest( "multiple active delegated tooltips", function() {
 });
 
 }( jQuery ) );
+
+// http://bugs.jqueryui.com/ticket/11272
+test ( "remove radio button conflict with live Region and tooltip content", function () {
+	expect( 3 );
+	var content = "<div id='content'>" +
+			"<input type='radio' name='hobby' checked='checked'><label>option 1</label>" +
+			"<input type='radio' name='hobby'><label>option 2</label>" +
+		"</div>",
+		stripped_content = "<div>" +
+				"<input checked=\"checked\" type=\"radio\"><label>option 1</label>" +
```

Comment:
Why are you using single quotes above and escaped double quotes here? Be consistent.